docs/prow/test-clusters.md

```diff
@@ -0,0 +1,34 @@
+# Prow test clusters
+
+This document gathers information about test clusters that are build by Prow jobs. All test clusters (a.k.a. workload clusters) are build in sap-kyma-prow-workloads project.
+
+
+## Cluster authorization
+
+### Permissions
+
+Kyma developers are gathered in `kyma_developers@sap.com` Google Group administrated by [Neighbors team](https://github.com/orgs/kyma-project/teams/prow/members?utf8=%E2%9C%93&query=role%3Amaintainer). All the group permission are managed in IAM, where the `kyma_developers@sap.com` group has `kyma_developer` role assigned so that its members can access test clusters and VMs in read-only mode.
```

Comment:
```suggestion
Kyma developers are gathered in the `kyma_developers@sap.com` Google Group administrated by the [Neighbors team](https://github.com/orgs/kyma-project/teams/prow/members?utf8=%E2%9C%93&query=role%3Amaintainer). All the group permissions are managed in IAM, where the `kyma_developers@sap.com` group has the **kyma_developer** role assigned so that its members can access test clusters and VMs in read-only mode.
```